pytest_html/plugin.py

```diff
@@ -154,8 +154,8 @@ def _appendrow(self, result, report):
             html.td(test_id, class_='col-name'),
             html.td('{0:.2f}'.format(time), class_='col-duration'),
             html.td(links_html, class_='col-links'),
-            html.td(additional_html, class_='extra')],
-            class_=result.lower() + ' results-table-row'))
+            html.td(additional_html, class_='extra', id=test_id)],
+            class_=result.lower() + ' results-table-row', id=test_id))
```

Comment:
We shouldn't have two elements with the same id.

 > The id global attribute defines a unique identifier (ID) which must be unique in the whole document.

From: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id